Packs/ctf01/ReleaseNotes/1_0_24.md

```diff
@@ -0,0 +1,7 @@
+<!-- 
+#### Integrations
+
+##### Cortex XDR - IR CTF
+
+- No changes related directly to this integration.
+ -->
```

Comment:
```suggestion
No changes related directly to this integration.
```

If no changes were made, then why do we need to add this?

Reply:
since changes were made to ctf01 dependent pack (CoreIRApiModule ) a new version needs to be released.